ql/src/java/org/apache/hadoop/hive/ql/session/OperationLog.java

```diff
@@ -73,10 +74,10 @@ public OperationLog(String name, File file, HiveConf hiveConf) {
       opLoggingLevel = LoggingLevel.UNKNOWN;
     }
 
+    isRemoveLogs = hiveConf.getBoolVar(HiveConf.ConfVars.HIVE_TESTING_REMOVE_LOGS);
```

Comment:
I would rather keep these configurations independent:
- `HIVE_TESTING_REMOVE_LOGS`
- `HIVE_SERVER2_HISTORIC_OPERATION_LOG_ENABLED`

And if any of them are set to true, then do not remove the logs

Reply:
Done, check the HIVE_SERVER2_HISTORIC_OPERATION_LOG_ENABLED when hive not in test. Thanks for the review!